manimlib/scene/scene.py

```diff
@@ -719,6 +719,9 @@ def combine_movie_files(self):
         )
         with open(file_list, 'w') as fp:
             for pf_path in partial_movie_files:
+                if os.name == 'nt':
+                    pf_path = pf_path.replace('\\', '/')
```

Comment:
The ffmpeg seems not able to recognize the backslash in the partial_movie_files_list.txt.